Update the command to package the local source into a Docker image.

as6325400 opened the pull request:
When packaging my local source into a package, I encountered an error because I didn't execute the command in the parent directory of DOMjudge. It took me some time to realize the issue when I initially filled in the absolute path for `<domjudge source directory>`.

I believe it should be mentioned in the README to execute the command in the parent directory of DOMjudge, or the changes made in this PR could help avoid this issue.

vmcj commented on docker/README.md:
````diff
@@ -263,8 +263,10 @@ where `version` is the DOMjudge version to create the images for, e.g. `5.3.0`.
 ```bash
-  tar --exclude-vcs -czf <path to domjudge-packaging>/docker/domjudge.tar.gz <domjudge source directory>
-  cd <path to domjudge-packaging>/docker
+  dpkp=<path to domjudge-packaging>
+  dj=<domjudge source directory>
+  tar --exclude-vcs -czf $dpkp/docker/domjudge.tar.gz -C $(dirname "$dj") $(basename "$dj")
+  cd $dpkp/docker
````
```suggestion
  dj_packaging_dir=<path to domjudge-packaging>
  dj_source_dir =<domjudge source directory>
  tar --exclude-vcs -czf "${dj_packaging_dir}/docker/domjudge.tar.gz" -C $(dirname "$dj_source_dir") $(basename "$dj_source_dir")
  cd ${dj_packaging_dir}/docker"
```

Lets better explain which directories are used, did not test if this still works.

vmcj commented:
@as6325400 can you squash and improve the commit message by explaining why this was needed? I'll merge afterwards.

as6325400 commented:
> @as6325400 can you squash and improve the commit message by explaining why this was needed? I'll merge afterwards.

is this okay Thank?

nickygerritsen commented:
The message is missing *why* we need this. If you can add that, we will merge.

as6325400 commented:
> The message is missing _why_ we need this. If you can add that, we will merge.

I’ve added the missing message. I also noticed there was an extra quotation mark (") in my previous command, and I’ve fixed that as well.
